fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -139,6 +145,36 @@ export const LocationSuggest = forwardRef<HTMLInputElement, Props>(
       getLocationSuggest,
     ]);
 
+    const loadInitialLocation = useCallback(async () => {
+      if (!initialValue || initialLocation?.id.value === initialValue) {
+        return;
+      }
+
+      if (!client) {
+        return;
+      }
+      const { data } = await client.query({
+        ...(client && { client }),
+        fetchPolicy: 'no-cache',
+        query: LOCATION,
+        variables: { id: initialValue },
+      });
+
+      if (!data.location) {
+        return;
+      }
+
+      setInitialLocation(data.location);
+
+      if (!selectedLocationId) {
+        setSelectedLocationId(data.location.id.value);
+      }
+    }, [initialValue, client, selectedLocationId, initialLocation]);
+
+    useEffect(() => {
+      loadInitialLocation();
+    });
```

Comment:
This will prevent the effect from re-running after the initial render:

```suggestion
    }, []);
```

Linting may ask for this which is fine too:


```suggestion
    }, [loadInitialLocation]);
```